plugins/appwrite/cli_token.go

```diff
@@ -0,0 +1,56 @@
+package appwrite
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func CLIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.CLIToken,
+		DocsURL:       sdk.URL("https://appwrite.io/docs/command-line"),
```

Comment:
Is there a more specific documentation link that explains the credential itself, instead of the CLI itself as a whole?